test/testingHelpers.js

```diff
@@ -84,8 +89,64 @@ function mockResponse(sandbox) {
   return res
 }
 
+function mockIDTokenFactory(reason) {
+  return JSON.stringify({
+    // eslint-disable-next-line no-underscore-dangle
+    aud: reason.audience === true ? 'not-pa' : googleHelpers.__get__('PA_CLIENT_ID'),
```

Comment:
This part here where you do the boolean comparison `x === true ? something : other thing` instead of simply `x ? something : other thing` is non-idiomatic (i.e. it's unusual to see in JS). 
Was there a particular reason that it needed to compare against `true` explicitly?

Reply:
Very good point! I'm not sure why I checked for being equal to true, lol.